doc/sphinx/proofs/writing-proofs/rewriting.rst

```diff
@@ -552,90 +517,78 @@ the conversion in hypotheses :n:`{+ @ident}`.
    Example: The term :g:`fun n : nat => S n + S n` is not reduced by :n:`hnf`.
 
 .. note::
- The :math:`\delta` rule only applies to transparent constants (see :ref:`vernac-controlling-the-reduction-strategies`
- on transparency and opacity).
+ The :math:`\delta` rule only applies to transparent constants
+ (see :ref:`controlling-the-reduction-strategies` on transparency and opacity).
+
+.. tacn:: simpl {? @delta_reductions } {? {| @reference_occs | @pattern_occs } } {? @occurrences }
 
-.. tacn:: cbn
-          simpl
-   :name: cbn; simpl
+   .. insertprodn reference_occs pattern_occs
 
-   These tactics apply to any goal. They try to reduce a term to
-   something still readable instead of fully normalizing it. They perform
+   .. prodn::
+      reference_occs ::= @reference {? at @occs_nums }
+      pattern_occs ::= @one_term {? at @occs_nums }
+
+   Reduces a term to
+   something still readable instead of fully normalizing it. It performs
    a sort of strong normalization with two key differences:
 
-   + They unfold a constant if and only if it leads to a :math:`\iota`-reduction,
+   + It unfolds constants only if they lead to an :math:`\iota`-reduction,
```

Comment:
I'm surprised by this change of "a" to "an". To me the "i" in the beginning of "iota" is used as a consonant (like "y") rather than a vowel.

Reply:
In English, the "i" is pronunced just like the pronoun "I".

Reply:
OK! I didn't know that! Thanks for teaching me!